src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,361 @@
-REGIONS	ec2.ap-northeast-2.amazonaws.com	ap-northeast-2
-REGIONS	ec2.ap-northeast-1.amazonaws.com	ap-northeast-1
-REGIONS	ec2.us-east-1.amazonaws.com	us-east-1
-REGIONS	ec2.sa-east-1.amazonaws.com	sa-east-1
-REGIONS	ec2.us-west-1.amazonaws.com	us-west-1
-REGIONS	ec2.us-west-2.amazonaws.com	us-west-2
+
+
+#### linux
+
+```bash
+$ curl -sSL https://github.com/kubernetes/kops/releases/download/1.5.1/kops-linux-amd64 -O
+$ chmod +x kops-darwin-amd64
+$ sudo mv kops-darwin-amd64 /usr/local/bin/kops
 ```
 
-## Download and Unpack Kubernetes
+For more information see the official [kops installation guide](https://github.com/kubernetes/kops/blob/master/docs/aws.md).
 
-First, make a directory to hold the Kubernetes release files:
+## Validate kops is installed
 
+```bash
+$ kops version
+Version 1.5.1
 ```
-$ mkdir my-first-cluster
-$ cd my-first-cluster
+
+## Install kubectl if you haven't done so yet
+
+```bash
+$ curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+$ chmod +x kubectl
+$ sudo mv kubectl /usr/local/bin/kops
 ```
 
-See [Kubernetes Versions](https://deis.com/docs/workflow/installing-workflow/system-requirements/#kubernetes-versions) under System Requirements and download a Kubernetes release that is compatible with Deis Workflow, and extract the archive on your machine.
 
-This archive has everything that you need to launch Kubernetes. It's a fairly large archive, so it may take some time to download:
+## Setup your AWS account
+
+#### Setup an IAM user for kops
 
+In order to build clusters within AWS we'll create a dedicated IAM user for
+`kops`.  This user requires API credentials in order to use `kops`.  Create
+the user, and credentials, using the [AWS console](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html).
+
+The `kops` user will require the following IAM permissions to function properly
+
+ - AmazonEC2FullAccess
+     - This is used to deploy to instances in EC2
+ - AmazonRoute53FullAccess
+     - This is used so kops can automatically create friendly DNS records for your cluster resources
+ - AmazonS3FullAccess
+     - This is used to store meta configuration about your cluster. We will need read/write here to use S3 as a virtual filesystem in kops.
+ - IAMFullAccess
+     - This is used because kops will create new IAM users for some of it's resources. Those resources will have permissions managed securely by kops.
+ - AmazonVPCFullAccess
+     - This used to create a VPC which serves as the foundation of all networking components in kops. Without a VPC, kops wouldn't be able to deploy any resources dependent on a network.
+
+#### Create the IAM user from the command line
+
+```bash
+$ curl -O https://raw.githubusercontent.com/kubernetes/kops/master/hack/new-iam-user.sh
+$ sh new-iam-user.sh <group> <user>
+$ aws iam list-users
```

Comment:
Actually I see we continue to reference `aws` throughout the doc, so its installation instructions should probably be restored.